modules/axp/Animal/lib/Mouse.pm

```diff
@@ -11,7 +11,7 @@ with 'Animal';
 
 =head1 NAME
 
-Mouse - The mouse goes ˢᑫᵘᵉᵃᵏ
+Mouse - The mouse goes squeak
```

Comment:
I had to make this change for Exercise 2&mdash; it didn't like the utf-8 character set when it first ran `disttest`.